### ISSUE TITLE
Blue Lines - Mario Party 2 (Light Out Minigame)

### ISSUE THREAD

**Issue by KenanUchiha28** [gonetz/GLideN64#2592]
![135771738-4b91ef4b-9e8d-4cd9-8fc0-0b5b41905528](https://user-images.githubusercontent.com/91834977/136122459-d151d04d-2518-4c12-b524-bf5c6834f4af.png)
Is there a way to get rid of these blue lines? I'm using MupenPlusNext on Retroarch.

**Comment by Xii-Nyth** [comment]
> possibly related to #2248 , it seems the game renders the top part of the fence as a 2d background image while the front is 3d and polygon misalignment occuring @gonetz ? maybe im wrong

Yeah since thats in front of the 3d parts and its the only part effected that would make sense, but since theres no aa on it so it must be 2d. Im pretty sure the problem in perfect dark is caused by the fact that most vertices are not properly connected in models, I went fixing them in blender but havent been able to test out if it fixes this because I can't figure out how to re import the object files.

**Comment by Jj0YzL5nvJ** [comment]
They are halos. #1924

**Comment by weinerschnitzel** [comment]
Is this with accurate texture coordinates, and is it visible in native resolution or multiples of native resolution when using accurate texture coords?

Pretty sure that was the silver bullet for ogre battle text. I would guess the same here.
